packages/expect/src/matchers.ts

```diff
@@ -87,10 +86,10 @@ const matchers: MatchersObject = {
     return {actual: received, expected, message, name: 'toBe', pass};
   },
 
-  toBeCloseTo(received: number, expected: number, precision?: number = 2) {
+  toBeCloseTo(received: number, expected: number, precision: number = 2) {
     const secondArgument = arguments.length === 3 ? 'precision' : null;
     const isNot = this.isNot;
-    const options = {
+    const options: any = {
```

Comment:
`MatcherHintOptions` type which is exported from `jest-matcher-utils`

Reply:
This has been updated